.github/workflows/tests.yaml

```diff
@@ -33,4 +33,4 @@ jobs:
 
             -   uses: "ramsey/composer-install@v1"
 
-            -   run: vendor/bin/paratest ${{ matrix.path }} --colors --verbose
+            -   run: vendor/bin/phpunit ${{ matrix.path }} --colors --verbose
```

Comment:
@samsonasik Btw, where can we setup number of parallel processes? Is there some config for paratest?

Reply:
it was using `-p4` ref https://github.com/rectorphp/rector-src/pull/551/files#diff-95557d53bf91069e59d82b9d8fcfaf52ec84a762858983edd5ef3c9b3e4c8191

Reply:
Let's try :) 

Still it seems we could decopule one more domain package to offload the rules tests, e.g. composer related content, mysql-to-mysqli or similar.